media-libs/opusfile/files/http-fix-compilation-with-LibreSSL.patch

```diff
@@ -0,0 +1,114 @@
+From 8486e83cdba2d8c537edeacfc33eabb7ebf9f0e5 Mon Sep 17 00:00:00 2001
+From: Stefan Strogin <stefan.strogin@gmail.com>
+Date: Thu, 27 Sep 2018 04:53:29 +0300
+Subject: [PATCH] http: fix compilation with LibreSSL
+
+Fixes: #2327
+---
+ src/http.c | 22 +++++++++++-----------
+ 1 file changed, 11 insertions(+), 11 deletions(-)
+
+diff --git a/src/http.c b/src/http.c
+index 8ba9b28..b88613c 100644
+--- a/src/http.c
++++ b/src/http.c
+@@ -1530,7 +1530,7 @@ static long op_bio_retry_ctrl(BIO *_b,int _cmd,long _num,void *_ptr){
+   return ret;
+ }
+ 
+-# if OPENSSL_VERSION_NUMBER<0x10100000L
++# if OPENSSL_VERSION_NUMBER<0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
I think this may break with libressl-2.8.

Reply:
Thanks for the remark Michał, I'll try to build it with libressl-2.8 and see if it breaks.

Reply:
It successfully builds with Libressl-2.8.0 with or without the patch. The methods like BIO_{g,s}et_data(), BIO_set_init() and others are implemented in libressl-2.8.0 but the old approach works as well.